src/more-infos/more-info-media_player.html

```diff
@@ -96,7 +100,7 @@
       <div class='controls layout horizontal justified'
            hidden$='[[computeHideSelectSource(isOff, supportsSelectSource)]]'>
         <iron-icon class="source-input" icon="mdi:login-variant"></iron-icon>
-        <paper-dropdown-menu class="flex source-input" label-float label='Source'>
+        <paper-dropdown-menu class="flex source-input" label-float label='Source' vertical-align="bottom">
```

Comment:
You can use the `dynamic-align` property instead  which will auto-pick direction with more space.

Reply:
Thanks, I changed this.